lib/versioner/staggered.go

```diff
@@ -48,23 +48,12 @@ func NewStaggered(folderID, folderPath string, params map[string]string) Version
 		cleanInterval = 3600 // Default: clean once per hour
 	}
 
-	// Use custom path if set, otherwise .stversions in folderPath
-	var versionsDir string
-	if params["versionsPath"] == "" {
-		versionsDir = filepath.Join(folderPath, ".stversions")
-		l.Debugln("using default dir .stversions")
-	} else if filepath.IsAbs(params["versionsPath"]) {
-		l.Debugln("using dir", params["versionsPath"])
-		versionsDir = params["versionsPath"]
-	} else {
-		versionsDir = filepath.Join(folderPath, params["versionsPath"])
-		l.Debugln("using dir", versionsDir)
-	}
+	versionsDir := ".stversions"
```

Comment:
Silent deprecation of the custom directory? I wonder what the best way to handle this is...